src/lib/requestmanager.ts

```diff
@@ -51,18 +74,18 @@ export class RequestManager {
 		err: Error | null | undefined,
 		result?: NetworkOpResult,
 	) {
-		const entry = this.#entries.get(id)
-		if (entry) {
+		const request = this.#requestsById.get(id)
+		if (request) {
 			trace(`InvokeId ${id} found -> call callback`)
-			this.#entries.delete(id)
+			this.#requestsById.delete(id)
```

Comment:
this is the other place where you delete the requestById

Reply:
Done in https://github.com/bacnet-js/client/commit/1079027100e655907586130d21883785fb6e73ec